vortex/swap-to-vanilla-launcher.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/bash
+VORTEX_PREFIX=~/.vortex-linux/compatdata/pfx;
+LOADERS=();
+GAMELAUNCHERS=();
+GAMELOADERS=();
+GAMENAMES=();
+INSTALLDIRECTORY=();
+printf "%s\n" "UPDATE: Updating loaderlibrary.vdf to the most recent version";
+rm -f ~/.pikdum/steam-deck-master/vortex/loaderlibrary.vdf
+wget -q https://raw.githubusercontent.com/SirStig/Steam-deck-tests/main/vortex/loaderlibrary.vdf -P ~/.pikdum/steam-deck-master/vortex/
```

Comment:
**suggestion:** Rather than updating on every run, we can keep updates explicit behind the 'Update' shortcut.

```suggestion
```